apps/vr-tests/package.json

```
@@ -34,7 +34,7 @@
     "@fluentui/react-make-styles": "^9.0.0-alpha.59",
     "@fluentui/react-menu": "^9.0.0-alpha.68",
     "@fluentui/react-provider": "^9.0.0-alpha.69",
-    "@fluentui/react-slider": "^9.0.0-alpha.0",
+    "@fluentui/react-switch": "^9.0.0-alpha.0",
```

Don't you need both?

```suggestion
    "@fluentui/react-slider": "^9.0.0-alpha.0",
    "@fluentui/react-switch": "^9.0.0-alpha.0",
```